jsk_panda_robot/jsk_panda_teleop/script/start_master_side.sh

```diff
@@ -0,0 +1,24 @@
+
+function init_ros_ip () {
+    export ROS_MASTER_URI=http://133.11.216.227:11311
```

Comment:
we should write our IP settings on README.md and information on where should we fix, if we change IP address of devices.

Reply:
Setting IP address manually is removed in #1543 and #1544